terraform/addons/byo-firehose-logging-destination/target-account/variables.tf

```diff
@@ -18,7 +18,7 @@ variable "firehose_status_name" {
 variable "firehose_audit_name" {
   type        = string
   description = "firehose delivery stream name for Fleet audit logs"
-  default     = ""
+  default     = "fleet_audit"
```

Comment:
I am guessing the defeault is now to _always_ include audit rather than optionall add?  The other policy changes seem to match that.